PURL-SPECIFICATION.rst

```diff
@@ -247,8 +247,9 @@ Use these rules for percent-encoding and decoding ``purl`` components:
 - the '#', '?', '@' and ':' characters must NOT be encoded when used as
   separators. They may need to be encoded elsewhere
 
-- the ':' ``scheme`` and ``type`` separator does not need to and must NOT be encoded.
-  It is unambiguous unencoded everywhere
+- The colon ':' separator between ``scheme`` and ``type`` MUST NOT be encoded.
+  For example, in the PURL snippet ``pkg:npm`` the colon ':' MUST NOT be encoded,
+  and the PURL snippet ``pkg%3Anpm`` is invalid.
 
```

Comment:
@gernot-h   @pombredanne   Consider adding at the top of the file, perhaps as a new one-line paragraph following the current first paragraph, something along the lines of the following:

> This specification uses RFC 2119 (https://datatracker.ietf.org/doc/html/rfc2119), as clarified in RFC 8174 (https://datatracker.ietf.org/doc/html/rfc8174), for the interpretation of certain terms, e.g., ``MUST NOT``.

Or perhaps a slight modification to the example provided by RFC 2119:

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119), as clarified in [RFC 8174](https://datatracker.ietf.org/doc/html/rfc8174).

(Note that the core spec currently contains a great deal of language that will need to be modified to implement RFC 2119/8174.)